bnelson777/id.ly
# IL-148: App Icons

To test: 
Run this branch and verify that the app icon matches the following: 
![icon-512](https://user-images.githubusercontent.com/22675571/40880363-397995da-6664-11e8-8c66-40bc5074d52d.png)
